drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -716,6 +716,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     var renderView = DragAndDropTemplates(configuration);
 
     var $element = $(element);
+    element = $element[0];  // Works around this Studio bug: https://github.com/edx/edx-platform/pull/11433
```

Comment:
I think that restoring this line is an alternative to https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/153 . We should pick one approach.

CC @ciuin

Reply:
Not fussed here -- @mtyaka which method would you prefer?

Reply:
I slightly prefer the restored approach, but am fine with either.

I still think the root cause is a bug in edx-platform. It sometimes initializes the xblock with a jquery object and sometimes with a plain DOM element.

Reply:
Ok, since @ciuin's https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/153 has already been approved, let's go with that.

Reply:
> I still think the root cause is a bug in edx-platform. It sometimes initializes the xblock with a jquery object and sometimes with a plain DOM element.

It is, but unfortunately [when we've tried to fix it at the platform level](https://github.com/edx/edx-platform/pull/11433), [new things broke](https://github.com/edx/edx-platform/pull/11674).